styles/styles.css

```diff
@@ -122,6 +122,37 @@
     transition: right 0.2s ease-out;
 }
 
+#outline-delay-slider {
+    -webkit-appearance: none;
+    margin: 18px 0;
+    width: 100%;
+}
+
+#outline-delay-slider:focus {
+  outline: none;
+}
+
+#outline-delay-slider::-webkit-slider-runnable-track {
```

Comment:
I like the styling that you have done, but it somehow doesn't really look in-place to me.

Would you mind playing around with color and size a bit?
Maybe try the handle in the button color (with a similar focus indicator) and the bar in a dark grey?

Reply:
I will play a bit.

Reply:
Something like this:

![outline-autohide-delay-dark](https://user-images.githubusercontent.com/6791792/47888508-22a2bb80-de45-11e8-91c1-67d1b6b6d9e1.png)
![outline-autohide-delay-light](https://user-images.githubusercontent.com/6791792/47888514-29313300-de45-11e8-871a-9de8ec94e602.png)

Reply:
Yes, I like this much better. Thanks.